src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs

```diff
@@ -1,23 +1,29 @@
 using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Runtime.InteropServices.WindowsRuntime;
 using Uno.Extensions;
 using Uno.Extensions.Specialized;
-using Windows.Foundation;
 using Windows.Foundation.Collections;
-using Windows.UI.Xaml;
-using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Controls.Primitives;
 using Windows.UI.Xaml.Data;
-using Windows.UI.Xaml.Documents;
 using Windows.UI.Xaml.Input;
 using Windows.UI.Xaml.Media;
 
 namespace Windows.UI.Xaml.Controls
 {
 	public partial class Pivot : ItemsControl
 	{
+		private static class PivotHeaderItemSelectionStates
+		{
+			public const string Disabled = "Disabled";
+			public const string Unselected = "Unselected";
+			public const string UnselectedLocked = "UnselectedLocked"; // TODO: This is not currently in use.
```

Comment:
Anyone knows what this is for?

Reply:
No, but we may end up replacing this code with the official pivot code when available.